tests/fsharp/typecheck/sigs/neg01.bsl

```diff
@@ -3,10 +3,4 @@ neg01a.fsi(24,8,25,7): typecheck error FS0913: Types cannot contain nested type
 
 neg01a.fs(22,8,23,7): typecheck error FS0913: Types cannot contain nested type definitions
 
-neg01b.fs(2,13,2,14): typecheck error FS0039: The value, constructor, namespace or type 'X' is not defined. Maybe you want one of the following:
-
-   z
-
-   A
-
-   B
+neg01b.fs(2,13,2,14): typecheck error FS0039: The value, constructor, namespace or type 'X' is not defined.
```

Comment:
I'm assuming this is not giving suggestions because it's under the `minStringLengthForSuggestion` of 3?

Reply:
yes we ignore small identifiers of length 1 or 2 because they are hardly a spelling mistake